### DISCUSSION TITLE
Sensor that finds the cheapest non-contiguous agile slots

### DISCUSSION THREAD

**Discussion by pyriel800** [BottlecapDave/HomeAssistant-OctopusEnergy#507] [Ideas]
Hi,

I'm not sure if this has come up so please delete/point me at as necessary.

I think it would be useful to have a sensor that finds the lowest set of agile slots in a non-contiguous fashion over a desired number of hours in addition to the current contiguous block fashion.

This is based on the fact that there have been 30 minute slots that in total would add up to a number of hours in total but would be missed by the calculation of a continuous block for the same time.

I appreciate that this may not be a good idea so please feel free to tell me if I'm wrong!

Thanks!

**Comment by BottlecapDave** [comment]
Hi, this is already available by setting intermittent instead of continuous when creating a target rate sensor. Unless I'm misunderstanding something?

**Reply by pyriel800** [reply]
Nope. You're absolutely right and I should have read a little more deeply! Thanks for the reply and setting me straight!
